.github/workflows/pytest-cpu.yaml

```diff
@@ -0,0 +1,80 @@
+name: Pytest
+on:
+  pull_request: {}
+  workflow_dispatch: {}
+jobs:
+  pytest:
+    timeout-minutes: 30
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          - container: mosaicml/pytorch:1.11.0_cpu-python3.8-ubuntu20.04
+            markers: 'not daily and not gpu and not vision and not remote and not doctest'
```

Comment:
I don't think we want to turn off remote tests, right?

Reply:
They should run only daily. Daily will come in a separate PR

Reply:
hm... I feel like we shouldnt be hitting AWS every commit... open to suggestions tho

Reply:
I suppose its fine if it runs daily. It doesn't really seem like a problem to run on every commit though.

Reply:
up to you